summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/batch/BatchedStore.scala

```diff
@@ -93,14 +105,14 @@ trait BatchedStore[K, V] extends scalding.Store[K, V] { self =>
 
   protected def sumByBatches[K1, V: Semigroup](ins: TypedPipe[(Timestamp, (K1, V))],
     capturedBatcher: Batcher,
-    commutativity: Commutativity): TypedPipe[((K1, BatchID), (Timestamp, V))] = {
+    commutativity: Commutativity): TypedPipe[(LTuple2[K1, BatchID], (Timestamp, V))] = {
```

Comment:
Your right, it doesn't make any sense, the type leaks out. I could make the tuple type public itself maybe. Any changes to this will break the public api alas, though not in a way almost anyone would ever use, but still possible.